### PR TITLE
Added Stone,paper & scissor in super_views.py

### PR DESCRIPTION
**Pull request by haxxorsid**
Created function& defined `def sps`. 
Added `("/(?P<s0rp0rs>stone|paper|scissor)$", self.sps)` in self.routes.
Made few changes in string _HELP_TEXT_ to show new commands and change few other commands for better interface. 
Added **/stone /scissor /paper**. Changed commands like _/s(earch), /i(mage)_ to _/search,/image_.
Tested and working.
[Screenshot](http://goo.gl/ti2wHc)

### CODE REVIEW

**Comment by montemarcaldera** [comment]
You deleted the help function

**Comment by haxxorsid** [comment]
@montemarcaldera  Nope it is still there I think.

File: src/views/super_views.py [hunk 1]
```diff
@@ -27,24 +28,32 @@ def even_or_odd(self, message=None, match=None, to=None):
...
+    def sps(self, message=None, match=None, to=None): 
+	list= ['stone','paper','scissor'] #Created a list for random.
+	txt = random.choice(list)
+	is_choice = match.group("s0rp0rs")
+	if (list.index(txt) + 1 == list.index(is_choice)) or (list.index(txt) -2 == list.index(is_choice)) : # Index of your choice is more than auto generated choice, hence You win. Exception: Stone-Scissor case.
+		return TextMessageProtocolEntity("You: %s\nBOT: %s\nYou won!." % (is_choice, txt), to=message.getFrom())
+	elif (list.index(txt) == list.index(is_choice) + 1) or (list.index(txt) == list.index(is_choice) - 2): # Index of your choice is less than auto generated choice, hence You lose. Exception: Stone-Scissor case.
+		return TextMessageProtocolEntity("You: %s\nBOT: %s\nYou lost ." % (is_choice, txt), to=message.getFrom())
+	else:
+		return TextMessageProtocolEntity("You: %s\nBOT: %s\nIt's a tie." % (is_choice, txt), to=message.getFrom())
```
Comment (ghost) [inline src/views/super_views.py:42]: Where is the help function?

**Comment by montemarcaldera** [comment]
The help function isn't there lol

**Pull request closed without merging**